### ISSUE TITLE
Unable to specify a path to configuration file that contains a space

### ISSUE THREAD

**Issue by finnianr** [cherokee/webserver#1265]
I am unable to configure this file with following command
```
sudo cherokee-admin --target=$HOME/.config/Hex\ 11\ Software/myching-server/cherokee.conf
```
Using quotation marks instead of escaping the spaces does not make any difference. I get the following error message in config tool: 
```
The configuration file /home/finnian/.config/Hex was not found.
```

**Comment by skinkie** [comment]
Interesting. I'll investigate where this comes from. Does the regular server work for you? cherokee --config=$HOME/.config/Hex\ 11\ Software/myching-server/cherokee.conf

**Comment by finnianr** [comment]
Yes, the regular server does work with spaces in the path, so the issue is specific to the admin program

**Comment by finnianr** [comment]
I also tried using -C instead of --target with the same outcome
